src/main/java/com/github/kokorin/jaffree/ffmpeg/DesktopCaptureInput.java

```diff
@@ -0,0 +1,115 @@
+/*
+ *    Copyright  2019 Denis Kokorin
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package com.github.kokorin.jaffree.ffmpeg;
+
+import com.github.kokorin.jaffree.OS;
+
+import java.awt.*;
+
+/**
+ * Most of the information comes from https://trac.ffmpeg.org/wiki/Capture/Desktop
+ * <p>
+ * TODO list:
+ * - Screen selection when multiscreen?
+ * - Audio Capture?
+ * - Warn if framerate is not set (like FrameInput)
+ * - Call ffmpeg to return list of devices? list of screens?
+ */
+public class DesktopCaptureInput extends BaseInput<DesktopCaptureInput> implements Input {
+    private final boolean WINDOWS_USE_GDI = true;
```

Comment:
Oops, I meant it to be a static configuration.
In fact, I think it should disappear in the long run but I'm still not convinced which one is best.  GDI allows cropping but DirectShow seems the way to go for audio. 
If we could mix GDI for video and Directshow for audio, I think it would be perfect, but I have to test further. That would prevent other sources such as webcams however, but we're in a Desktop Capture context anyway.
If the conclusion is that we need to keep both modes, I'll add a setter and use an enum indeed.